setup.py

```diff
@@ -7,7 +7,7 @@ def readme():
 
 
 setup(name='gestalt-cfg',
-      version='1.0.9',
+      version='1.1.0',
```

Comment:
Since it appears we're undoing how we previously handled things, we should move to v2.

Reply:
Yes this will move to v2 but I want the merge for the other MR happen first.

Reply:
Updated as well to v2